cypress/integration/unit/agendaitem-changes.spec.js

```diff
@@ -1,24 +1,32 @@
-/* global context, beforeEach, it, cy */
+/* global context, before, beforeEach, it, cy */
 // / <reference types="Cypress" />
 import agenda from '../../selectors/agenda.selectors';
 import actionModal from '../../selectors/action-modal.selectors';
 
 context('Agendaitem changes tests', () => {
+  before(() => {
+    cy.resetCache();
+  });
+
   beforeEach(() => {
     cy.server();
     cy.login('Admin');
   });
+  const agendaURL = '/vergadering/5EBA48CF95A2760008000006/agenda/f66c6d79-6ad2-49e2-af55-702df3a936d8/agendapunten';
+  const approvalTitle = 'Goedkeuring van het verslag van de vergadering van vrijdag 22-11-2019.';
```

Comment:
Klopt het dat deze hardcoded in de testset zit? - just checking.

Reply:
bestaande data idd. Wijzigt niet zomaar